DataFormats/TrackerCommon/interface/TrackerTopology.h

```diff
@@ -19,9 +19,11 @@ class TrackerTopology {
     unsigned int layerStartBit_;
     unsigned int ladderStartBit_;
     unsigned int moduleStartBit_;
+    unsigned int doubleStartBit_;
     unsigned int layerMask_;
     unsigned int ladderMask_;
     unsigned int moduleMask_;
+    unsigned int doubleMask_;
```

Comment:
@adewit , we have [couple of warning in IBs](https://cmssdt.cern.ch/SDT/cgi-bin/buildlogs/el8_amd64_gcc11/CMSSW_13_2_X_2023-06-18-0000/DataFormats/TrackReco) due to this change. Can you please provide a fix with correct default values for these newly added member here https://github.com/cms-sw/cmssw/blob/master/DataFormats/TrackReco/test/testHitPattern.cpp#L16

Reply:
Hi @smuzaffar - since this member is not used for phase 0 or 1 I will just 0-initialize it in testHitPattern.cpp. Given the unit tests apparently didn't reveal this problem, what test(s) should I run locally to make sure everything is now OK?